clrsvsim/simulator.py

```diff
@@ -217,6 +217,9 @@ def modify_copy_number(input_bam, output_bam, chrom, start, end, ref_genome, rat
     Returns:
         A tuple of: (number of reads in the region, number of reads written in the region, number of split reads written)
     """
+    random_seed = int(random_seed) if random_seed else 2016
```

Comment:
This seems a bit odd, I'd expect no random seed to be used by default?  (Or at least put this in the default args.)

Reply:
ah, that's a great point! Fixed here and everywhere.